app/lib/search/query_builder.rb

```diff
@@ -240,6 +241,12 @@ def suggest_query
       { "suggest" => "spelling_with_highlighting" }
     end
 
+    def boost_fields_query
```

Comment:
Hey, do we want to take advantage of something like this here?:
https://github.com/alphagov/finder-frontend/pull/3028/files
https://github.com/alphagov/govuk-helm-charts/pull/1040
(we could add an ALLOW_BOOST_FIELDS or something like that)

Reply:
Comment resolved async. We'll probably move this config to the licence transaction [JSON file in Specialist Publisher](https://github.com/alphagov/specialist-publisher/blob/main/lib/documents/schemas/licence_transactions.json) after we've tested it as that seems a better place if this feature is wanted long term (more work to implement so not doing it now).